releasenotes/notes/remove-deprecated-code-in-0.22.0-139fa09ee0cc6df9.yaml

```diff
@@ -0,0 +1,36 @@
+
+upgrade:
+
+  - |
+    Removed the argument `qubit_channel_mapping` in :class:`qiskit.transpiler.passes.calibration.rzx_builder.RZXCalibrationBuilder`
+    
+    old:
+   	  # Create an RZXCalibrationBuilder object with the qubit_channel_mapping argument 
+      builder = RZXCalibrationBuilder(qubit_channel_mapping=[[0, 1]])
+    new:
+      # Create a target object for the Aer simulator
+      target = providers.aer.QasmSimulator().target()
+      # Create an RZXCalibrationBuilder object using the target object
+      builder = RZXCalibrationBuilder(target=target)
+      
+  - |
+    Replaced the argument `qobj[Qobj]` in :meth:`qiskit.providers.aer.QasmSimulator.run()` with `qcirc[QuantumCircuit]`
+    
+    old:
+      # Transpile the circuit for the target simulator
+      simulator = QasmSimulator()
+      transpiled_circuit = transpile(qc, simulator)
+      # Assemble the transpiled circuit into a Qobj
+      qobj = assemble(transpiled_circuit, shots=1024)
+      # Run the simulation
+      job = simulator.run(qobj)
+    new:
+      # Transpile the circuit for the target simulator
+      simulator = QasmSimulator()
+      transpiled_circuit = transpile(qc, simulator)
+      # Run the simulation
+      job = simulator.run(transpiled_circuit, shots=1024)
```

Comment:
I would also keep this second section very simple, the deprecation was already explained in the original PR (see reno [here](https://github.com/Qiskit/qiskit/pull/5128/files#diff-1c7f90e0c4c55483edffcd15b5f3ef23dbfeb8ed05b5267be17d44369425bc9e)). So here, I would get rid of the "old" section and just mention that the deprecated `qobj` input has been removed and an example with the new use (in a code block as explained above).

Reply:
Definitely will do and thank you so much for your continuous feedback. Being new to the OS contributions, all your suggestions were invaluable for me.